engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/MigrateProcessInstanceConcurrentTest.java

```diff
@@ -353,6 +353,20 @@ public void shouldContinueMigratedInstanceWithTimerAfter() {
             .timer(TimerIntent.TRIGGER, timerCreated.getValue())
             .key(timerCreated.getKey()));
 
+    final var migrationRejection =
+        RecordingExporter.processInstanceMigrationRecords(ProcessInstanceMigrationIntent.MIGRATE)
+            .withProcessInstanceKey(processInstanceKey)
+            .onlyCommandRejections()
+            .getFirst();
+
+    assertThat(migrationRejection)
+        .hasRejectionType(RejectionType.INVALID_STATE)
+        .hasRejectionReason(
+            """
+            Expected to migrate process instance '%s' but active element with id 'A' has a boundary event. \
+            Migrating active elements with boundary events is not possible yet."""
```

Comment:
:wrench: We adjusted the test case to match the new behavior. However, this changes the intention of the test case.

Instead of changing the test case, we could annotate the test with `@Ignore` until boundary events are supported.